### ISSUE TITLE
Fix cyclic import dependency in challenges/aws_utils.py

### ISSUE THREAD

**Issue by Ram81** [Cloud-CV/EvalAI#3146]
### Description

Currently, we can't import `Challenge` model or any method from `challenges/utils.py` in [challenges/aws_utils.py](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/aws_utils.py) because `aws_utils.py` is being imported in `challenges/model.py`. If we try to import any util methods or models from `challenges/utils.py` or `challenges/models.py` in `challenges/aws_utils.py` it fails because of cyclic imports.

### Fix

Refactor `challenges/models.py` to fix cyclic imports

**Comment by burnerlee** [comment]
@Ram81 can I pick up this issue? I'll make a quick PR

**Comment by RishabhJain2018** [comment]
Sure @burnerlee !
So, I am unassigning you from #2994 .

**Comment by burnerlee** [comment]
oh can i take up only one issue at a time?

**Comment by RishabhJain2018** [comment]
Yes, you can finish this one and then pick that one :)

**Comment by burnerlee** [comment]
oh ,cool

**Comment by burnerlee** [comment]
￼Opened PR #3148 to fix the issue. Please review @RishabhJain2018 @Ram81

**Comment by burnerlee** [comment]
Please also reassign me #2994

**Comment by Ram81** [comment]
@burnerlee it seems like you didn't test the changes by fixing imports in `aws_utils.py`. After fixing cyclic imports we want to place all `from .models` import at the start of the file in `aws_utils.py` and not in the methods. 

If you check [this](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/aws_utils.py#L356) you'll see the workaround for handling cyclic import is still there. If you move all such imports to start of the file we still see the same issue. If you're still interested you can pick this issue after you are done with your current task.

**Comment by burnerlee** [comment]
yeah, I'll pick this issue, I've my all PRs updated now and made a PR for the issue in eval-cli
